core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -155,6 +163,18 @@ public UserDTO addUser(UserDTO user, Optional<String> overwriteUserConfirmationC
 		return userDTO;
 	}
 
+	private void addMandatoryGoals(User userEntity)
```

Comment:
A simpler way to write this would be:
```
	private void addMandatoryGoals(User userEntity)
	{
		activityCategoryService.getAllActivityCategories().stream().filter(c -> c.isMandatoryNoGo())
				.forEach(c -> addGoal(userEntity, c));
	}

	private void addGoal(User userEntity, ActivityCategoryDTO category)
	{
		userEntity.getAnonymized()
				.addGoal(BudgetGoal.createNoGoInstance(ActivityCategory.getRepository().findOne(category.getID())));
	}
```